### ISSUE TITLE
Libreoffice color distorted and some applications fail to run

### ISSUE THREAD

**Issue by Chuanhuan** [pelya/xserver-xsdl#138]
![Screenshot_20210430-152956_XServer XSDL.jpg](https://user-images.githubusercontent.com/30125119/116663288-6929f000-a986-11eb-9667-75e1bee2d135.jpg)

This is one of my issues. Also, another bug I want to report is that whenever I execute "flameshot" app for taking screenshot. It just closed however it works fine in vncserver.

**Comment by zanfix** [comment]
Yep, this happens with libreoffice, with xsdl 24bit color video mode
